### PR TITLE
AMP-29009: Make ActivityImporter testable

### PR DESCRIPTION
**Pull request by vchihai**

### CODE REVIEW

File: amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/activity/ActivityImporter.java [hunk 1]
```diff
@@ -146,77 +155,92 @@ private void init(Map<String, Object> newJson, boolean update, String endpointCo
...
+    
+    /**
+     * Import or updates activity.
+     * @param activityId
+     */
+    private void importOrUpdateActivity(Long activityId) {
+        boolean oldActivityDraft = false;
         try {
-            // initialize new activity
-            InterchangeUtils.getSessionWithPendingChanges();
```
Comment (developster) [inline amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/activity/ActivityImporter.java]: getSessionWithPendingChanges is no longer called

File: amp/src/test/java/org/digijava/kernel/ampapi/endpoints/activity/AMPRequestRule.java [hunk 2]
```diff
@@ -1,5 +1,8 @@
 package org.digijava.kernel.ampapi.endpoints.activity;
 
+import static org.mockito.Matchers.any;
```
Comment (developster) [inline amp/src/test/java/org/digijava/kernel/ampapi/endpoints/activity/AMPRequestRule.java]: Please revert this file.

File: amp/src/test/java/org/digijava/kernel/ampapi/endpoints/activity/TestActivityService.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        this.tmContext = tmContext;
+    }
+    
+    @Override
+    public boolean isActivityStale(Long ampActivityId, Long activityGroupVersion) {
+        return stale;
+    }
+    
+    @Override
+    public void doWithLock(Long ampActivityId, Long ampTeamMemId, Runnable runnable) {
```
Comment (developster) [inline amp/src/test/java/org/digijava/kernel/ampapi/endpoints/activity/TestActivityService.java]: What is the purpose of this code?

File: amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/activity/ActivityImporter.java (hunk 1 above)
Comment (vchihai) [reply amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/activity/ActivityImporter.java]: The flush mode will be set in doWithLock method

**Pull request closed without merging**